common-features/adaptive-rendering.md

```diff
@@ -65,6 +66,40 @@ Three breakpoints define the rendering options as follows:
 **Dimensions** | up to 500px | 501px to 768px | over 768px |
 **Rendering** | The popup is rendered as a fullscreen action sheet. | The popup is rendered as an action sheet docked to the bottom of the screen. | The popup is rendered as an animation container docked to the main element of the component. |
 
+## Customize the Default Adaptive Breakpoints
+
+You can customize the [above-listed default adaptive breakpoints](#rendering-specifics) at the root level by configuring the [`<TelerikRootComponent>`]({%slug rootcomponent-overview%}). To specify your desired breakpoints:
+
+1. Wrap the content of the `<TelerikRootComponent>` (`@Body` and potentially other elements) in `<ChildContent>` tag.
+1. Add the `<RootComponentSettings>` component inside the [`<TelerikRootComponent>`]({%slug rootcomponent-overview%}).
+1. Add the `<RootComponentAdaptiveSettings>` component inside the `<RootComponentSettings>` tag and configure its properties:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type | Description |
+| ----------- | ----------- | ----------- |
+| `Small` | `int` | The upper boundary of the small threshold. Sets the `max-width` of the small media query in `px`. |
+| `Medium` | `int` | The upper boundary of the medium threshold. Sets the `max-width` of the medium media query in `px`.|
```

Comment:
These have default values that we should mention. See the `AdaptiveBreakpoint` enum.